pkg/util/validation/errors.go

```diff
@@ -243,6 +243,24 @@ func (e *histogramBucketLimitExceededError) Error() string {
 	return fmt.Sprintf("native histogram bucket count exceeded for metric (limit: %d) metric: %.200q", e.limit, formatLabelSet(e.series))
 }
 
+// histogramSchemaLimitExceededError is a ValidationError implementation for samples with native histogram
+// exceeding the min schema limit.
+type histogramSchemaLimitExceededError struct {
+	series []cortexpb.LabelAdapter
+	limit  int
+}
+
+func newHistogramSchemaLimitExceededError(series []cortexpb.LabelAdapter, limit int) ValidationError {
+	return &histogramSchemaLimitExceededError{
+		series: series,
+		limit:  limit,
+	}
+}
+
+func (e *histogramSchemaLimitExceededError) Error() string {
+	return fmt.Sprintf("native histogram min schema exceeded for metric (limit: %d) metric: %.200q", e.limit, formatLabelSet(e.series))
```

Comment:
native histogram schema is not a limit so it is weird to say exceeded. Let's just call it `nativeHistogramSchemaInvalidError`

Reply:
InvalidSchema error makes more sense.

I was trying to see what error Prometheus returns. But was surprised to see that Prometheus doesn't check for minSchema at scrape time.